public/index.html

```diff
@@ -36,15 +37,15 @@ <h2 class="main-header">Records</h2> <button class="filter-menu-button">All</but
     <ul class="disc-list">
       <li>
         <article class="disc-card">
-          <img class="cover" src="images/disc_image.webp" alt="Bridges to Babylon album cover" width="343" height="343">
+          <img class="cover" src="images/disc_image.webp" alt="Album cover by artist" width="343" height="343">
```

Comment:
Ya os di feedback sobre estos alts que no describen nada, no volváis a ponerlos.

Reply:
Cambiado de nuevo y implementado código para que cree los alts dinámicamente según el h2 de la card